timesketch/api/v1/resources.py

```diff
@@ -1368,6 +1369,50 @@ def post(self, sketch_id):
                 annotations, status_code=HTTP_STATUS_CODE_CREATED)
 
 
+class AnalyzerPipelineResource(ResourceMixin, Resource):
```

Comment:
Maybe we want to extend this to also accept an analyzer name? So we can run a specific analyzer instead of all.

Reply:
cool, SG